languages/go/query.go

```
@@ -94,8 +97,7 @@ func (q *Query) Next() (*map[string]interface{}, error) {
 			defer q.ffiQuery.Delete()
 			return nil, nil
 		case QueryEventDebug:
-			// TODO
-			return nil, fmt.Errorf("Polar debugger is not yet implemented in Go.")
+			err = q.handleDebug(ev)
```

Do you need to return the error here?

---

Ah yeah I think so.

---

No, I don't it gets returned below.